.castor/docker.php

```diff
@@ -23,6 +23,7 @@
 use function Castor\open;
 use function Castor\run;
 use function Castor\variable;
+use function Castor\yaml_parse;
```

Comment:
can you bump the requirement on castor to 0.15 ?

Reply:
Already present in root castor file